.github/workflows/operate-ci-build-reusable.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       # Build: maven artifacts and Docker images
       - name: Build Maven
         run: |
-          mvn -f operate clean deploy -P -docker -DskipTests=true -DaltStagingDirectory=${{ github.workspace }}/staging-${{ env.BRANCH_NAME }} -DskipRemoteStaging=true -Dmaven.deploy.skip=true
+          mvn clean deploy -B -T1C -P -docker -DskipTests -DskipChecks -DaltStagingDirectory=${{ github.workspace }}/staging-${{ env.BRANCH_NAME }} -DskipRemoteStaging=true -Dmaven.deploy.skip=true
```

Comment:
I don't think that it is worth to do that. There are more modules which operate needs. Clients, protocol, etc.

Reply:
They are not that many, the full list contains:
- zeebe-client-java
- zeebe-bpmn-model
- zeebe-protocol
- zeebe-protocol-jackson
- zeebe-test-container
- zeebe-build-tools

No need to move them under another module, we just need a module that declares them

Reply:
@houssain-barouni, what about the transitive (Zeebe) dependencies? For example, the `zeebe-client-java` depends on `zeebe-gateway-protocol-impl`, etc.

Reply:
Ah I see, with `-pl zeebe-lib` they would be part of the maven build tree.

Reply:
Hmm, I think you could just do -pl, and it will build dependencies or? 

I actually did it on purpose to build everything to always make sure we have a stable code basis all time, and I didn't feel that it makes a huge problem. But happy to discuss this further.

At some point, we likely want to have also one common ci to build everything and likely all fast unit tests for all components (or at least the ones that have changed related to dependents). On main we can run more integration, e2e tests etc. this would allow us faster iteration (reducing build/test times and reducing time to merge etc)

Reply:
I would agree with building everything always to make sure it works.